src/LanguageServer.Engine/CompletionProviders/CompletionProvider.cs

```diff
@@ -94,13 +94,18 @@ protected virtual bool HandleTriggerCharacters(string triggerCharacters, Project
                 throw new ArgumentNullException(nameof(projectDocument));
 
             // Replace any characters that were typed to trigger the completion.
-            if (triggerCharacters != null)
+            if (!String.IsNullOrEmpty(triggerCharacters))
```

Comment:
```suggestion
            if (!string.IsNullOrEmpty(triggerCharacters))
```

Reply:
Sorry, just saw this one - I have to stop reviewing these on my phone; it’s too easy to lose track of where I’m up to.